plugin/pkg/admission/admit/admission.go

```diff
@@ -1,58 +0,0 @@
-/*
-Copyright 2014 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package admit
-
-import (
-	"io"
-
-	"k8s.io/apiserver/pkg/admission"
-)
-
-// Register registers a plugin
-func Register(plugins *admission.Plugins) {
-	plugins.Register("AlwaysAdmit", func(config io.Reader) (admission.Interface, error) {
```

Comment:
Can't just drop this name or we break any invocation using it. Can deprecate it (and probably should, along with some others)

Reply:
I see.